app/views/wizards/claims/add_mentor_wizard/_no_results_step.html.erb

```diff
@@ -0,0 +1,12 @@
+<% content_for :page_title, t(".page_title", trn: wizard.steps[:mentor].trn, contextual_text:) %>
```

Comment:
This is coming out at "No results found for ‘%{trn}’ - Enter a teacher reference number (TRN) - Add mentor"
I'm guessing you're passing in "Enter a teacher reference number (TRN)" as part of the caption in `edit.html.erb`. 
It should just me "Add mentor" ("Add mentor - %{school_name}" for support)